### ISSUE TITLE
OpenCV2 integration not working

### ISSUE THREAD

**Issue by FrancoisPerez** [wyolum/TouchSelfie#26]
I am not sure to understand how this is supposed to work but I can’t see any preview method implemented on the opencv wrapper.
Whenever I try to use it, the image is not showing.

**Comment by osbock** [comment]
yeah, no preview in cv2 mode. Not sure it's working at all right now, but
you shuld be able to hack it together.

**Comment by patz2009** [comment]
I'm actually looking at this myself, and wondering if cv2 is going to work at any point. As of right now, when I launch the photobooth I immediately get an error that the piCamera isn't present, when I'd like to use an alternative camera that should be able to be accessed via cv2.
